reclass/storage/common.py

```diff
@@ -13,6 +13,17 @@ def nodes(relpath, name):
         # no mangling required
         return relpath, name
 
+    @staticmethod
+    def composed_nodes(relpath, name):
+        if relpath == '.' or relpath == '':
+            # './' is converted to None
+            return None, name
+        parts = relpath.split(os.path.sep)
+        if parts[0].startswith("_"):
```

Comment:
@epcim this is the code to ignore compose_node when path starts with `_`

Notice how it ONLY ignores `_` when it's at the beginning of the path

`nodes/_generated/mysql.yml` - subfolder is ignored
`nodes/prod/_generated/mysql.yml` - subfolder is NOT ignored